src/Common/src/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -484,6 +486,119 @@ private static SizeAndAlignment ComputeInstanceSize(MetadataType type, int count
             return result;
         }
 
+        public override ValueTypePassingCharacteristics ComputeValueTypePassingCharacteristics(DefType type)
+        {
+            ValueTypePassingCharacteristics result = ComputeHomogeneousFloatAggregateCharacteristic(type);
```

Comment:
What is the expected behavior when somebody calls this on a reference type? Just return false?